readme.md

```diff
@@ -1,28 +1,484 @@
 #ES6
 
+## Learning Objectives
+
+- Compare/contrast features of ES5 and ES6
+- Explain when to use var vs let vs const
+- Use default parameters and arrow functions
+- Use template literals to interpolate variables and strings
+- Compare ES6 classes w/ ES5 prototypes
+- Use import/export syntax with modules
+
```

Comment:
Maybe add a LO relating to deconstruction...
- Use deconstruction to extract values from objects and arrays